dynawo/sources/Models/Modelica/Dynawo/Types.mo

```diff
@@ -66,5 +68,9 @@ package Types "Standard types for electrical variables"
   // Percent
   type Percent = Real (unit = "100") "Percent quantity";
 
+  // Impedance and admittance
+  type Impedance = Real (unit="ohms") "Impedance";
```

Comment:
Given that Impedance and Admittance are supposed to be complex, could we instead create types Resistance, Reactance, Conductance and Susceptance ?

Reply:
@bongrainmat do you think it is better to have types Impedance, Admittance or Resistance, Reactance, Conductance, Susceptance ?